src/styles/blocks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
You can use just - https://developer.mozilla.org/ru/docs/Web/CSS/background
To combine them